std_msgs/Bool approach_charging_station
---
std_msgs/Float64 range
std_msgs/Bool success
